Source/Particles/Collision/BackgroundMCCCollision.H

```diff
@@ -0,0 +1,77 @@
+/* Copyright 2021 Modern Electron
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef WARPX_PARTICLES_COLLISION_BACKGROUNDMCCCOLLISION_H_
+#define WARPX_PARTICLES_COLLISION_BACKGROUNDMCCCOLLISION_H_
+
+#include "Particles/MultiParticleContainer.H"
+#include "CollisionBase.H"
+#include "MCCProcess.H"
+
+#include <AMReX_REAL.H>
+#include <AMReX_GpuContainers.H>
+
+#include <string>
+
+class BackgroundMCCCollision
+    : public CollisionBase
+{
+public:
+    template <typename T>
+    using VectorType = amrex::Gpu::ManagedVector<T>;
```

Comment:
Discussed off-line: This could be removed, and we would simply use `amrex::Vector` or something similar.